engine/src/main/resources/settings.ini

```diff
@@ -4,7 +4,7 @@ fullscreen=false
 controlType=1
 sfxVol=1.0
 musicVol=0.25
-canSellEquippedItems=false
+canSellEquippedItems=true
```

Comment:
Whyyyy

Reply:
Because this is file that should certainly not be in git, but rather generated by gradle/groovyw/whatever. Also, I'm quite sure that I changed that option back to default afterwards?